RecoLocalCalo/HcalRecProducers/src/ZdcSimpleReconstructor.cc

```diff
@@ -60,7 +62,13 @@ void ZdcSimpleReconstructor::produce(edm::Event& e, const edm::EventSetup& event
   
   if (det_==DetId::Calo && subdet_==HcalZDCDetId::SubdetectorId) {
     edm::Handle<ZDCDigiCollection> digi;
-    e.getByToken(tok_input_,digi);
+    e.getByToken(tok_input_castor,digi);
+     
+     if(digi->size() == 0) {
+       e.getByToken(tok_input_hcal,digi);
+       if(digi->size() != 0) std::cout<<"No ZDC info in castorDigis, using hcalDigis. This is normal for all Run1 datasets"<<std::endl;
+       else std::cout<<"No ZDC info in either castorDigis or hcalDigis"<<std::endl;
```

Comment:
same comments here as in 
https://github.com/cms-sw/cmssw/pull/13882/files#r58400048